rdkit/Chem/ChemUtils/TemplateExpand.py

```diff
@@ -234,22 +233,25 @@ def ConstructSidechains(suppl, sma=None, replace=True, useAll=False):
         # atoms only:
         matches = mol.GetSubstructMatches(patt)
         if matches:
-          tmp = []
-          for match in matches:
+          tmp = [0] * len(matches)
+          for i, match in enumerate(matches):
             smi = Chem.MolToSmiles(mol, True, rootedAtAtom=match[0])
```

Comment:
@IchiruTake  See Greg's comment about the second argument.

Reply:
Ok, I have accepted the changes, I will put it in my remind list to check